ui-grid/ui-grid.d.ts

```diff
@@ -3766,6 +3766,14 @@ declare module uiGrid {
          * You may specify one of the sortingAlgorithms found in the rowSorter service.
          */
         sortCellFiltered?: boolean;
+        /**
+         *(optional) An array of sort directions, specifying the order that they should cycle through as 
```

Comment:
I don't think sortDirectionCycle?: Array<IUiGridConstants> is right, as it should be an array including uiGridConstants.ASC, uiGridConstants.DESC and/or null, while IUiGridConstants is the interface that contains these constants, not the constants themselves. When I try and use sortDirectionCycle, I get:
```
  Types of property 'sortDirectionCycle' are incompatible.
    Type 'string[]' is not assignable to type 'IUiGridConstants[]'.
      Type 'string' is not assignable to type 'IUiGridConstants'.
```